vdirsyncer/storage/dav.py

```diff
@@ -1,7 +1,15 @@
 # -*- coding: utf-8 -*-
 
+from __future__ import print_function
```

Comment:
On Sat, Mar 26, 2016 at 11:48:43AM -0700, Hugo Osvaldo Barrera wrote:
> Do we need this line? Isn't this just for unsupported python2 versions?

Is python2 really unsupported? I see just a warning about dropping its
support in the future. And also tests are running on python2 (in
addition to python3).

-- 
Best Regards,
Marek Marczykowski-Górecki
Invisible Things Lab
A: Because it messes up the order in which people normally read text.
Q: Why is top-posting such a bad thing?